socnet/templates/blog/subscriptions.html

```
@@ -33,25 +33,25 @@ <h3 class="list-group-item text-center">
           class="avatar-thumbnail d-flex align-items-center justify-content-center me-2"
         >
           {% if subscription.image %}
-            <img src="{{ subscription.image.url }}" loading="lazy" class="rounded" />
+            <img class="rounded" loading="lazy" src="{{ subscription.image.url }}" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)